app/src/main/java/org/simple/clinic/bp/BloodPressureRepository.kt

```diff
@@ -67,12 +67,15 @@ class BloodPressureRepository @Inject constructor(
   }
 
   fun updateMeasurement(measurement: BloodPressureMeasurement): Completable {
-    return Completable.fromAction {
-      val updatedMeasurement = measurement.copy(
-          updatedAt = Instant.now(utcClock),
-          syncStatus = SyncStatus.PENDING
-      )
+    val updatedMeasurement = measurement.copy(
+        updatedAt = Instant.now(utcClock),
+        syncStatus = SyncStatus.PENDING
+    )
+    //TODO [Encounter] Uncomment to enable
```

Comment:
Please explain, Uncomment to enable what?

Reply:
This is throw-away code. It is removed in the future commits. 
This was added to not break existing functionality while developing.
Do you want me to rebase edit this commit still?